TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -104,14 +104,14 @@ public function initializeArguments()
         $this->registerTagAttribute('alt', 'string', 'Specifies an alternate text for an image', true);
         $this->registerTagAttribute('ismap', 'string', 'Specifies an image as a server-side image-map. Rarely used. Look at usemap instead', false);
         $this->registerTagAttribute('usemap', 'string', 'Specifies an image as a client-side image-map', false);
-        // @deprecated since 2.0 use the "image" argument instead
-        $this->registerArgument('asset', 'TYPO3\Media\Domain\Model\AssetInterface', 'The image to be rendered - DEPRECATED, use "image" argument instead', false);
+        // @deprecated since 2.1 use "asset" argument instead
+        $this->registerArgument('image', 'TYPO3\Media\Domain\Model\ImageInterface', 'The image to be rendered - DEPRECATED, use "asset" argument instead', false);
```

Comment:
@kitsunet See earlier comments for discussion. Basically since the image view helper supports any assets now. It felt weird creating a almost exact duplicate view helper that accepts assets instead of only images. The view helper doesn't determine if it can generate a thumbnail, the thumbnail strategies do, so it would basically be two view helpers where one accepts image interface and one accepts asset interface.

For users there's no difference, it's all backwards compatible.

Don't sense a strong argument here, maybe you can elaborate?